<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModality::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>604</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>C++ Application</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>50</number>
          </property>
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QPushButton" name="enterBtn">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>ENTER</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="quitBtn">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>QUIT</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>275</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>275</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_4">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0">
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="themeLbl">
          <property name="styleSheet">
           <string notr="true">color: white;          
font-size: 20px;         
font-family: Arial;
font-weight: bold;
text-align: center;</string>
          </property>
          <property name="text">
           <string>CHOOSE A STORYLINE</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>44</number>
          </property>
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>APOLLO 18</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>DESERT OF UNDEAD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="logoutBtn">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>LOG OUT</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>QUIT </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="spacing">
             <number>20</number>
            </property>
            <item>
             <widget class="QLineEdit" name="userTxt">
              <property name="maximumSize">
               <size>
                <width>250</width>
                <height>30</height>
               </size>
              </property>
              <property name="placeholderText">
               <string>Enter Username</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="passTxt">
                <property name="maximumSize">
                 <size>
                  <width>202</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::EchoMode::Password</enum>
                </property>
                <property name="placeholderText">
                 <string>Enter Password</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_3">
                <property name="maximumSize">
                 <size>
                  <width>40</width>
                  <height>26</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Reveal/Hide Password</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset>
                  <normaloff>../../Downloads/padlock.png</normaloff>../../Downloads/padlock.png</iconset>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="loginBtm">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>Login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="signUpBtn">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>Sign up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>QUIT GAME</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>200</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_3">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="userNameTxt">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="placeholderText">
             <string>Username</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="PassTxt">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="CPassTxt">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::EchoMode::Password</enum>
            </property>
            <property name="placeholderText">
             <string>Confirm Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cornfirmBtn">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>Cornfirm</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: blue;  
                border: 2px solid white;  
                border-radius: 12px;       
                padding: 10px 24px;    
                font-size: 20px;             
                font-weight: bold;        
                text-align: center;   
                width: 200px;                
                height: 50px;      </string>
            </property>
            <property name="text">
             <string>BACK</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>250</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuC">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>C++</string>
    </property>
   </widget>
   <addaction name="menuC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
